src/super_gradients/training/utils/optimizer_utils.py

```diff
@@ -111,6 +112,12 @@ def build_optimizer(net: nn.Module, lr: float, training_params) -> optim.Optimiz
             "initial_lr training hyperparameter (i.e initial_lr={'backbone': 0.01, 'default':0.1})",
             DeprecationWarning,
         )
+    if training_params.finetune and isinstance(net, SupportsFineTune):
```

Comment:
Perhaps we should raise a warning if `if training_params.finetune` is True and model is not supporting this and not just silently continue?

Reply:
sure, 5321b2de50e9e66ff755d3663be919bdd50c112f